packages/flutter_tools/lib/src/linux/linux_doctor.dart

```diff
@@ -127,6 +127,8 @@ class LinuxDoctorValidator extends DoctorValidator {
       final _VersionInfo? version = installedVersions[kPkgConfigBinary];
       if (version == null || version.number == null) {
         messages.add(ValidationMessage.error(_userMessages.pkgConfigMissing));
+        // exit early because we cannot validate libraries without pkg-config
```

Comment:
nit: Start comment with a capital letter, and end with a period.

Reply:
done